### DISCUSSION TITLE
Always be imposter

### DISCUSSION THREAD

**Discussion by uwuUwuIo** [Impostor/Impostor#201] [Q&A] [answered]
Imposter

**Comment by uwuUwuIo** [comment]
I'd try but there's no code

**Comment by dimaguy** [comment]
Please use correct english grammar.
What do you mean with always being impost**o**r

**Comment by XtraCube** [comment]
Do you mean a way to make one player the Impostor in every game?

**Comment by dimaguy** [comment]
> 
> 
> If you mean a way to make one player the Impostor in every game, you would need to make a plugin that sets a certain player to be the impostor.

Let the issue opener explain what he wants to do. Its important not to spoonfeed

**Comment by AeonLucid** [comment, marked as answer]
Not possible, requires client modifications because of the host iirc.

**Comment by uwuUwuIo** [comment]
Sorry I'm 7

**Comment by edqx** [comment]
aren't we all
